src/backend/templates/schedule.hbs

```diff
@@ -539,9 +542,18 @@
                     $(sessionName).html(textArray.join(' ') + '...');
                 }
               })
+              roomCount++;
             });
+
             if(width < $(rooms).width()) {
-              $(rooms).width(width);
+              offset = ($(rooms).width() - width)/roomCount;
```

Comment:
Hey, @geekyd @Princu7 This is not the right approach to set width and current width every time the page loads with JS.
I hope you guys understand the effects of this on the performance of the app.
Proposed approach with some reasoning:
- Can we do this calculation on the server `calculation of offset` we already know the column width and we already know the number of columns. 
- After getting the `offset` as a handlebar variable, we can add it to HTML DOM element using CSS style. 
Please comment your suggestions @geekyd @Princu7.  :)